giftcard.adoc

```diff
@@ -1,46 +1,74 @@
-## Giftcard API
+== Giftcard API
 
-### Service Description
-The Giftcard API provides extra information about purchases made through iZettle with a giftcard.
+=== Description
+====
+Giftcard API provides extra information about purchases made through iZettle with a gift card.
+====
 
-### URL
+=== URL
 https://giftcard.izettle.com
 
-### Scopes
-The Giftcard API implements the following scopes:
-
+=== OAuth Scopes
 - `READ:PURCHASE`
 
-### Get Giftcard details
+'''
+
+==== Fetch gift card
 
 `GET organizations/self/giftcards/external/{uuid}`
 
-#### Permissions required
-`READ:PURCHASE`
+|===
+|Name|Type|In|Required/Optional|Description
+
+|`uuid`|string|path|required|Unique id as `UUID`.
```

Comment:
Is UUID or uuid?
In parameter tables, no special styling is needed. So I'd remove ``.
Do users know where they can find the uuid? If not, I'd add where to find it.
Usually when we enter credit card number at the payment, we're asked to enter security code. Not everyone knows what security code is and where to find it. So there's usually an instruction about where we can see the code on our cards.